jaxrs-tck-docs/userguide/src/main/jbake/content/using-examples.inc

```diff
@@ -21,6 +21,23 @@ mvn verify -DexcludedGroups="xml_binding"
 ----
 --
 
+Run the tests by excluding the security tests using the following command:
+
+--
+[source,oac_no_warn]
+----
+mvn verify -DexcludedGroups="security"
+----
+--
+
+Run the tests by excluding the servlet tests using the following command:
```

Comment:
It would be nice to specify whether the groups are comma separated ids, or how to set multiple exclude groups, as would be required for the core-profile.